src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -25,14 +26,16 @@
 
 @register(AlertRule)
 class AlertRuleSerializer(Serializer):
-    def __init__(self, expand=None):
+    def __init__(self, expand: Optional[List[str]] = None):
         self.expand = expand or []
 
-    def get_attrs(self, item_list, user, **kwargs):
+    def get_attrs(
+        self, item_list: Sequence[Any], user: Union[User, RpcUser], **kwargs: Any
```

Comment:
Even though _I_ know `item_list` is a list of `AlertRule`s, I wasn't able to type it that way:

```
src/sentry/api/serializers/models/alert_rule.py:32: error: Argument 1 of "get_attrs" is incompatible with supertype "Serializer"; supertype defines the argument type as "Sequence[Any]"  [override]
src/sentry/api/serializers/models/alert_rule.py:32: note: This violates the Liskov substitution principle
src/sentry/api/serializers/models/alert_rule.py:32: note: See https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides
```